operations/app/terraform/vars/test/locals.tf

```diff
@@ -10,6 +10,9 @@ locals {
     okta_redirect_url     = "https://prime-data-hub-XXXXXXX.azurefd.net/download"
     okta_base_url         = "hhs-prime.oktapreview.com"
     OKTA_scope            = "simple_report_dev"
+    RS_okta_base_url      = "reportstream.oktapreview.com"
+    RS_okta_redirect_url  = "https://prime-data-hub-XXXXXXX.azurefd.net/download"
+    RS_OKTA_scope         = "simple_report_dev"
```

Comment:
`RS_OKTA_scope` is `"reportstream_dev"`

Reply:
RS_OKTA_scope has been updated in the latest commit